dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Moderator/AzureContentSafetyModerator.cs

```diff
@@ -15,24 +15,16 @@ namespace Microsoft.TeamsAI.AI.Moderator
     public class AzureContentSafetyModerator<TState> : IModerator<TState> where TState : ITurnState<StateBase, StateBase, TempState>
     {
         private readonly AzureContentSafetyModeratorOptions _options;
-        private readonly AzureContentSafetyClient _client;
+        private readonly ContentSafetyClient _client;
 
         /// <summary>
         /// Constructs an instance of the moderator.
         /// </summary>
         /// <param name="options">Options to configure the moderator.</param>
-        /// <param name="logger">A logger instance.</param>
-        /// <param name="httpClient">HTTP client.</param>
-        public AzureContentSafetyModerator(AzureContentSafetyModeratorOptions options, ILogger? logger = null, HttpClient? httpClient = null)
+        public AzureContentSafetyModerator(AzureContentSafetyModeratorOptions options)
```

Comment:
Any sample change required?

Reply:
Good reminder, I forgot that. Updated three samples that use the moderator.